### ISSUE TITLE
Link not clickable on Whatsapp

### ISSUE THREAD

**Issue by erperejildo** [BranchMetrics/cordova-ionic-phonegap-branch-deep-linking#355]
I just realised that with some devices the link myapp.app.link is not clickable through Whatsapp, it's only clickable when it has http/https before.

Is there any way to fix this?

**Comment by ethanneff** [comment]
Hello,

Thank you for reaching out.

Unfortunately, this is a limitation of WhatsApp. The WhatsApp app will not recognize the `app.link` top level domain. This makes the links unclickable unless you add the `https/http` protocol. 

We have reached out to WhatsApp's dev team numerous times in the past year to get this fixed. We will continue to do so since it is still impacting partners like yourself.

In the meantime, you will either have to share your deep links with the `https/http` prefix or [switch your link domain](https://dev.branch.io/getting-started/link-domain-subdomain/guide/#custom-root-domain-branchcom) to a `.com` or other top level link domain. This will make your deep links clickable in Whatsapp.

I apologize for this inconvenience. I have updated the [Github readme](https://github.com/BranchMetrics/cordova-ionic-phonegap-branch-deep-linking#testing-supported-platforms) to reflect this linking behavior within WhatsApp.

Thanks,

**Comment by erperejildo** [comment]
Hi @ethanneff and thanks for your comment.

I tried Branch but as I explained a few days ago in [other post](https://github.com/EddyVerbruggen/Custom-URL-scheme/issues/234) I only can get the main url. If I need to redirect the user to myApp/options I can't get that info.

That's weird because with my Android phone it works well with every single link. Maybe the Whatsapp's team dev struggles as well with Android's team... who knows.

So apart from Branch solution is there any other way to get this working through Whatsapp (because being honest the 80% of our link will be there)?

Cheers
